index.html

```diff
@@ -1,23 +1,84 @@
 <!DOCTYPE html>
 <html lang="es">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Hostal las paisas</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<link rel="stylesheet" type="text/css" href="css/home.css">
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+	<title>Hostal las paisas</title>
 </head>
+
 <body>
-    <nav>
-         <!-- Navigation section -->
-    </nav>
-    <section>
-        <!-- Hero section, the cover for the page -->
-    </section>
-    <section>
-       <!-- comodities section -->
-    </section>
-    <footer>
-        <!-- footer section -->
-    </footer>
+	<nav>
+		<!-- Navigation section -->
+	</nav>
+	<section class="homepage-hero">
+		<!-- Hero section, the cover for the page -->
+		<div class="homepage-hero__layer"></div>
+		<article class="homepage-hero__article">
+			<div class="homepage-hero__headline">
+				<h1 id="" class="">Hostal</h1>
+				<h1 id="" class="">Las paisas</h1>
+			</div>
+			<div class="homepage-hero__divider"></div>
+			<span id="" class="homepage-hero__promotext">El paraiso te espera</span>
+		</article>
+	</section>
```

Comment:
Why are you adding content to the hero section?

Reply:
That is the code in the index.html in  our team ; i just copy my code.

Reply:
This code showed up as if it was yours, make sure that you do a pull and not a copy/paste from another branch, as this block will cause a conflict with the code on develop even while it is very similar

Reply:
fixed